docs/sdk/client-side/flutter/index.md

```diff
@@ -105,30 +116,135 @@ final client = BKTClient.instance;
 </TabItem>
 </Tabs>
 
-:::note
+:::info Default timeout
+
+The initialize process default timeout is 5 seconds.
 
-The initialize process immediately starts polling the latest evaluations from Bucketeer in the background using the interval `pollingInterval` configuration while the application is in the **foreground state**. When the application changes to the **background state**, it will use the `backgroundPollingInterval` configuration.
+Once initialization is finished, all the requests in the SDK use a timeout of 30 seconds.
 
 :::
 
-If you want to use the feature flag on Splash or Main views, and the user opens your application for the first time, it may not have enough time to fetch the variations from the Bucketeer server.
+If you want to use the feature flag on Splash or Main views, the SDK cache may be old or not exist and may not have enough time to fetch the variations from the Bucketeer server. For this case, we recommend using the `timeout` option in the initialize method. It will block until the SDK receives the latest user variations.
+
+:::caution Initialization Timeout error
 
-For this case, we recommend using the `timeout` option in the initialize method. It will block until the SDK receives the latest user variations.
+During the initialization process, errors **are not** related to the initialization itself. Instead, they arise from a timeout request, indicating the variations data from the server weren't received. Therefore, the SDK will work as usual and update the variations in the next [polling](flutter#polling) request.
+
+:::
 
 <Tabs>
 <TabItem value="dart" label="Dart">
 
 ```dart showLineNumbers
 /// It will unlock without waiting until the fetching variation process finishes
-int timeout = 1000;
+int timeout = 5000;
 
 await BKTClient.initialize(config: config, user: user, timeoutMillis: timeout);
-final client = BKTClient.instance;
+if (result.isSuccess) {
```

Comment:
Please update the code block with the following.

```suggestion
int timeout = 5000;
final result = await BKTClient.initialize(
  config: config, user: user, timeoutMillis: timeout);
if (result.isSuccess) {
  final client = BKTClient.instance;
  if (showNewFeature) {
    /// The Application code to show the new feature
  } else {
    /// The code to run if the feature is off
  }
} else {
  /// Handle the error when there is no cache or the cache is not updated
}
```

Reply:
done